report/src/presence_technologies/bluetooth.tex

```diff
@@ -0,0 +1,44 @@
+\section{Bluetooth} \label{sec:bluetooth}
+
+Another possible technology for presence detection is Bluetooth.
+Bluetooth is a common technology used in many devices such as smartphones, tablets, and laptops.
+Because it is a standardized technology, all devices that support Bluetooth can communicate with each other.
```

Comment:
We need a cite that it is standardized

Reply:
There is a cite. It's at the end of the section. The cite covers the whole section.

Reply:
A cite only covers a single paragraph